test/libsolidity/syntaxTests/license/license_whitespace_3.sol

```diff
@@ -0,0 +1,2 @@
+// SPDX-License-Identifier: GPL-3.0   
```

Comment:
I'd add a comment below saying that there's trailing whitespace here. Some people might have editors configured no to show it (github does not show it either) and just trim it without realizing.

I'd also rename it to `license_trailing_whitespace.sol`.